sigmastate/src/main/scala/org/ergoplatform/ErgoBox.scala

```diff
@@ -175,7 +184,7 @@ object ErgoBox {
 
   val allZerosModifierId: ModifierId = Array.fill[Byte](32)(0.toByte).toModifierId
 
-  def apply(value: Long,
+  def create(value: Long,
```

Comment:
Similarly to previous comment , consider to rename the method to testBox

Reply:
I replaced implicit `apply` with explicit `create` factory method to make more clear where the default constructor is used and where is not. 
Other changes is beyond the scope of this PR, which is adding test vectors.
Added TODO refactor.

Reply:
@aslesarenko "create" is not addressing context where this method should be used (testing only)

Also, better not to make ErgoBox case class, who knows if any descendant will appear for which it will be more convenient to use case-class form.

Reply:
moved this method to tests